recipes/trollsift/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "trollsift" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "4f67522224714e4c1ca612c82c5a1600c55fef879efbe1f30bcbe8f79e442ed0" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - trollsift
+
+about:
+  home: https://github.com/pytroll/trollsift
+  license: GPLv3+
+  license_family: GPL3
+  license_file: ''
+  summary: String parser/formatter for PyTroll packages
+  description: Filtering modules used in PyTroll packages
+  doc_url: http://trollsift.readthedocs.io
+  dev_url: https://github.com/pytroll/trollsift
+
+extra:
+  recipe-maintainers:
+    - djhoese
+    - mraspaud
+    - adybbroe
+    - pnuu
```

Comment:
@pnuu are you okay with being a maintainer on the conda forge feedstock?

Reply:
Yes, I'm okay with it.